PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -72,6 +73,8 @@ end
 local function CreateCamera()
 	local this = {}
 
+	this.cframe = workspace.CurrentCamera and workspace.CurrentCamera.CoordinateFrame or CFrame.new()
```

Comment:
The reasoning is that previously, the camera scripts would just work off the existing state of the camera object. If your VR offset is modifying that, your state is now invalid unless you roll it back somehow, which is prone to floating point precision errors and isn't very friendly if you need to mix the transforms in a different way. It's definitely necessary if we want reasonable VR code.